source/Playnite/GamesEditor.cs

```diff
@@ -746,6 +753,23 @@ public void SetHideGames(List<Game> games, bool state)
             }
         }
 
+        public void SetHdr(Game game, bool state)

```

Comment:
Should be called `SetHdrSupport` or something similar.

Reply:
I called it `SetHdrEnabled` to match the `IsHdrEnabled`

Reply:
done